README.md

```diff
@@ -1,4 +1,4 @@
-# prop-types <sup>[![Version Badge][npm-version-svg]][package-url]</sup>
```

Comment:
I'm not interested in a version badge in this style; the proper fix is to update the `npm-version-svg` URL below to be `http://versionbadg.es/airbnb/prop-types.svg`. Can you make that change instead?

Reply:
Sorry I wasn't being clear :-) the `[npm-version-svg]` is a reference to the matching line at the bottom of the markdown document, so that urls never need to go inline. This line should be reverted, and the URL at the bottom of the markdown document should be updated instead.

Reply:
Ah, copy that. Updated in the latest.

Reply:
There's still a diff here though; the effect of which is to make the version number no longer be clickable - prior to this PR, it links to the npm package, which is intended. Can you revert this line entirely, so that the only change is on line 50?